### ISSUE TITLE
tflite cannot fuse `BatchNormalization` in Keras Core as effectively as in the original Keras

### ISSUE THREAD

**Issue by james77777778** [keras-team/keras-core#824]
Let's take MobileNetV3Small as example

```python
import tensorflow as tf
from keras_core.applications.mobilenet_v3 import MobileNetV3Small  # keras-core
from tensorflow import keras  # keras

keras_core_model = MobileNetV3Small(
    input_shape=(224, 224, 3), minimalistic=True
)
keras_model = keras.applications.MobileNetV3Small(
    input_shape=(224, 224, 3), minimalistic=True
)

for model, name in [
    (keras_core_model, "keras_core"),
    (keras_model, "keras"),
]:
    tf_callable = tf.function(
        model.call,
        input_signature=[tf.TensorSpec((1, 224, 224, 3), tf.float32)],
        autograph=True,
        jit_compile=True,
    )
    tf_concrete_function = tf_callable.get_concrete_function()
    converter = tf.lite.TFLiteConverter.from_concrete_functions(
        [tf_concrete_function], tf_callable
    )
    # do quantization...
    converter.optimizations = [tf.lite.Optimize.DEFAULT]
    tflite_model = converter.convert()
    with open(f"{name}_model.tflite", "wb") as f:
        f.write(tflite_model)
```

The visualization from netron:
Keras Core vs. Keras:
![](https://user-images.githubusercontent.com/20734616/262847739-dceba686-ce2d-48d6-9110-900fe5142fa8.jpg) ![](https://user-images.githubusercontent.com/20734616/262847745-74f207c2-eb4c-4b4a-8595-f373f0b5e40c.jpg)

We can see that `BatchNormalization` was divided into `[Sub, Mul, Mul, Add]` compared to the fused op in the original Keras.

I believe the reason is that `BatchNormalization` in Keras Core is built using `ops`.
However, in the original Keras, `BatchNormalization` can switch to using `_fused_batch_norm` with `tf.compat.v1.nn.fused_batch_norm`

https://github.com/keras-team/keras/blob/94a1712027366cd626df4f1d77f25f918a18f456/keras/layers/normalization/batch_normalization.py#L978-L988

The unfused ops will reduce the inference speed. This issue makes Keras Core less suitable for tflite deployment.
